notebooks/problem-sets/multi-qubit_states.ipynb

```diff
@@ -0,0 +1,380 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 1\n",
+    "\n",
+    "<!-- ::: q-quiz(goal=\"multi-qubit-states-1\") -->\n",
+    "\n",
+    "<!-- ::: .question -->\n",
+    "\n",
+    "Which is an entangled quantum state?\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "1. $\\frac{|01\\rangle + |11\\rangle}{\\sqrt{2}}$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option -->\n",
+    "\n",
+    "2. $\\frac{|00\\rangle + |01\\rangle + |10\\rangle + |11\\rangle}{2}$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: .option(x) -->\n",
+    "\n",
+    "3. $\\frac{|01\\rangle + |10\\rangle}{\\sqrt{2}}$\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->\n",
+    "\n",
+    "<!-- ::: -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 2\n",
+    "\n",
+    "Create a `QuantumCircuit` that represents Control-Z gate using a cnot gate and single-qubit operations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "goals": [
+     {
+      "id": "multi-qubit-states-2"
+     }
+    ],
+    "grader_answer": "qc",
+    "grader_id": "textbook_practice_exam/multi-qubit_states/problem2"
```

Comment:
the `grader_id` should be in the form of `<event or exam>/<problem id>` so therefore this should be 
`textbook_practice_exam/problem2` or better yet to avoid possible conflict with other problems in `textbook_practice_exam` maybe `textbook_practice_exam/multi-qubit-problem2` would be a better `grader_id` value